### PR TITLE
first changelog

### PR DESCRIPTION
**Pull request by cgeorgilakis**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Changelog
+All notable eosc-kc changes of Keycloak will be documented in this file.
+For Keycloak upstream changelog please see https://www.keycloak.org/docs/latest/release_notes/index.html.
+Full Keycloak upstream jira issue can be shown if filtered by Fix version. For example [Keycloak jira issue for 15.0.2 version](https://issues.redhat.com/browse/KEYCLOAK-19161?jql=project%20%3D%20keycloak%20and%20fixVersion%20%3D%2015.0.2)
```
Comment (NicolasLiampotis) [inline CHANGELOG.md:4]: Replace with:
```
# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

For Keycloak upstream changes please see https://www.keycloak.org/docs/latest/release_notes/index.html.
Full Keycloak upstream jira issue can be shown if filtered by Fix version. For example [Keycloak jira issue for 15.0.2 version](https://issues.redhat.com/browse/KEYCLOAK-19161?jql=project%20%3D%20keycloak%20and%20fixVersion%20%3D%2015.0.2)

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
I would suggest the following changes:
1. Reverse order of releases so that latest release is shown first
2. Replace `github issue XX` with `EOSC-KC-XX`

See also inline comments

**Comment by cgeorgilakis** [comment]
> I would suggest the following changes:
> 
> 1. Reverse order of releases so that latest release is shown first
> 2. Replace `github issue XX` with `EOSC-KC-XX`
> 
> See also inline comments

I agree especially bullet 1 is based on https://keepachangelog.com/en/1.0.0/. I have made requested changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- new release created on tag
+
+## [v15.0.2-r1.0.2] - 5-10-2021
+### Fixed
+- Brokered SAML logins fail due to wrong InResponseTo in 15.x. [KEYCLOAK-19143](https://issues.redhat.com/browse/KEYCLOAK-19143)
+
+## [v15.0.2-r1.0.1] - 29-9-2021
+### Added
+- Support for SAML IdP federation
+- User reaccepting terms and conditions. [EOSC-KC-48](https://github.com/eosc-kc/keycloak/issues/48)
```
Comment (laskasn) [inline CHANGELOG.md:28]: Please add these as well:

Create a minimal realm representation and use it in Admin Console [EOSC-KC-93](https://github.com/eosc-kc/keycloak/issues/93)
Allow fetching of Identity Providers in pages and enhance Admin Console by using that [EOSC-KC-73](https://github.com/keycloak/keycloak/discussions/8608)

Comment (cgeorgilakis) [reply CHANGELOG.md]: I have previously added EOSC-KC-49 with description 'Terms and Conditions - periodically ask for reacceptance.' . I have changed description based on your comment. However, I believe that I should use term 'Terms and Conditions' that used in Keycloak documentation.

See my add about EOSC-KC-73.

**Pull request merged**